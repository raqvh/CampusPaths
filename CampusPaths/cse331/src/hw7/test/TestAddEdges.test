# Test we can load a graph then change the edges to have a new value
# If a new value is added, the weights of the previous edges should not have changed
# but this new edge should still be used to find the most cost effective path

LoadGraph csefriends CSEfriends.tsv

FindPath csefriends Raquel Duncan

# This should result in changing the most cost effective path from Raquel to Duncan
AddEdge csefriends Raquel Duncan 0.0

# Check 
FindPath csefriends Raquel Duncan