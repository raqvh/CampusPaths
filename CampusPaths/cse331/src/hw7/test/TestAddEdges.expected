# Test we can load a graph then change the edges to have a new value
# If a new value is added, the weights of the previous edges should not have changed
# but this new edge should still be used to find the most cost effective path

loaded graph csefriends

path from Raquel to Duncan:
Raquel to Duncan with weight 0.500
total cost: 0.500

# This should result in changing the most cost effective path from Raquel to Duncan
added edge 0.000 from Raquel to Duncan in csefriends

# Check 
path from Raquel to Duncan:
Raquel to Duncan with weight 0.000
total cost: 0.000
